cli/pack/cpio.go

```diff
@@ -0,0 +1,38 @@
+package pack
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func packCpio(relPaths []string, resFileName, packageFilesDir string) error {
+	filesBuffer := bytes.Buffer{}
+	filesBuffer.WriteString(strings.Join(relPaths, "\n"))
+
+	cpioFile, err := os.Create(resFileName)
+	if err != nil {
```

Comment:
Why do we fill filesBuffer before this check?

Reply:
Fixed. Sorry, the code is taken from cartridge-cli implementation.